OJP/OJP_Trips.xsd

```diff
@@ -703,12 +703,17 @@
 			<xs:documentation>Allowed values for a AccessFeature.</xs:documentation>
 		</xs:annotation>
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="lift"/>
+			<xs:enumeration value="elevator"/>
```

Comment:
The term *elevator" ist quite consistently used in the text of the NeTEx EPIAP document. In the code, however, the term *lift* is used (LiftEquipment). Shall we change the token back to lift?

If yes, then elevatorHasTactileButtons, etc., have do be adapted accordingly.

Reply:
I can live with it as it is. but other opinions are welcome.